src/main/java/org/aksw/simba/lemming/metrics/single/AvgVertexDegreeMetricResult.java

```diff
@@ -0,0 +1,56 @@
+package org.aksw.simba.lemming.metrics.single;
+
+public class AvgVertexDegreeMetricResult extends SimpleMetricResult {
```

Comment:
This class does not seem to be used :confused: 

I do not understand why the class has results for "addEdge" and "removeEdge". This does not fit to the overall workflow which foresees a single result object for each of these cases instead of one result object that stores everything.